docker-compose.yml

```diff
@@ -25,8 +25,10 @@ services:
   zipkin:
     image: jeqo/zipkin-kafka
     container_name: zipkin
+    hostname: zipkin
```

Comment:
anywhere setting hostname explicitly, comment why important. If important we should probably consider changing upstream docker-compose also..